deploy/platform_login.sh

```diff
@@ -16,11 +16,9 @@ main() {
       -u oauth2accesstoken \
       -p "$(gcloud auth print-access-token)"
   elif [[ "$PLATFORM" = "openshift" ]]; then
-    # sensible default for OPENSHIFT_URL port
-    if [[ -n "${OPENSHIFT_URL}" ]] && [[ "${OPENSHIFT_URL}" != *: ]]; then
-     OPENSHIFT_URL="${OPENSHIFT_URL}:8443"
+    if [[ -n "${OPENSHIFT_URL}" ]] && [[ ! "${OPENSHIFT_URL}" =~ :[[:digit:]] ]]; then
```

Comment:
would be helpful to comment what is being achieved here

Reply:
done, thanks